src/components/TagPicker/index.js

```diff
@@ -40,7 +40,6 @@ function TagPicker({selectedTag, tag, policyTags, policyRecentlyUsedTags, onSubm
     const initialFocusedIndex = useMemo(() => {
         if (isTagsCountBelowThreshold && selectedOptions.length > 0) {
             return _.chain(policyTagList)
-                .values()
```

Comment:
This is needed because `policyTagList` is object.
You can console log `initialFocusedIndex`. Removing this `initialFocusedIndex` is always -1.
Also please apply your original fix + my suggestion. This will do the job